packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -509,8 +511,19 @@ impl NeutronMsg {
     /// * **name** is a name of the schedule;
     /// * **period** is a period of schedule execution in blocks;
     /// * **msgs** is the messages that will be executed.
-    pub fn submit_add_schedule(name: String, period: u64, msgs: Vec<MsgExecuteContract>) -> Self {
-        NeutronMsg::AddSchedule { name, period, msgs }
+    /// * **blocker** is a blocker where schedule will be executed.
+    pub fn submit_add_schedule(
+        name: String,
+        period: u64,
+        msgs: Vec<MsgExecuteContract>,
+        blocker: u64,
```

Comment:
You've renamed `blocker` to `ExecutionStage` in go code. Let's do it everywhere. Also let's change a type of the argument in the method to `BlockerType` from [cron.rs file](https://github.com/neutron-org/neutron-sdk/pull/161/files#diff-6198a27407233de845943d780924eeb4c0b2e45d06260705b3b33a219b34362bR113).

Also do not forget to regenerate proto files

Reply:
Done